2021-02-04 17:59:50,331 INFO    MainThread:42208 [wandb_setup.py:_flush():70] setting env: {}
2021-02-04 17:59:50,331 INFO    MainThread:42208 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-02-04 17:59:50,331 INFO    MainThread:42208 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-04 17:59:50,331 INFO    MainThread:42208 [wandb_init.py:_log_setup():318] Logging user logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210204_175950-2dgvkb7m\logs\debug.log
2021-02-04 17:59:50,332 INFO    MainThread:42208 [wandb_init.py:_log_setup():319] Logging internal logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210204_175950-2dgvkb7m\logs\debug-internal.log
2021-02-04 17:59:52,130 INFO    MainThread:42208 [wandb_run.py:_console_start():1399] atexit reg
2021-02-04 17:59:52,131 INFO    MainThread:42208 [wandb_run.py:_redirect():1262] redirect: SettingsConsole.WRAP
2021-02-04 17:59:52,131 INFO    MainThread:42208 [wandb_run.py:_redirect():1297] Wrapping output streams.
2021-02-04 17:59:52,131 INFO    MainThread:42208 [wandb_run.py:_redirect():1313] Redirects installed.
2021-02-04 17:59:52,132 INFO    MainThread:42208 [wandb_run.py:_config_callback():663] config_cb None None {'LOG_DIR': '../logs', 'dataset': 'cars', 'sz_embedding': 512, 'sz_batch': 128, 'nb_epochs': 60, 'gpu_id': 0, 'nb_workers': 0, 'model': 'googlenet', 'loss': 'TripletPCA', 'optimizer': 'adamw', 'lr': 0.0001, 'weight_decay': 0.0001, 'lr_decay_step': 20, 'lr_decay_gamma': 0.5, 'alpha': 32, 'mrg': 0.1, 'IPC': None, 'warm': 1, 'bn_freeze': 1, 'l2_norm': 1, 'remark': ''}
2021-02-04 20:43:03,721 INFO    MainThread:42208 [wandb_run.py:_atexit_cleanup():1369] got exitcode: 0
2021-02-04 20:43:03,722 INFO    MainThread:42208 [wandb_run.py:_restore():1341] restore
2021-02-04 20:43:03,962 INFO    MainThread:42208 [wandb_run.py:_wait_for_finish():1492] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1263
  total_bytes: 1263
}

2021-02-04 20:43:05,965 INFO    MainThread:42208 [wandb_run.py:_wait_for_finish():1492] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 773756
  total_bytes: 773756
}

2021-02-04 20:43:07,049 INFO    MainThread:42208 [wandb_run.py:_show_summary():1647] rendering summary
2021-02-04 20:43:07,050 INFO    MainThread:42208 [wandb_run.py:_show_history():1685] rendering history
2021-02-04 20:43:07,051 INFO    MainThread:42208 [wandb_run.py:_show_files():1714] logging synced files
2021-02-04 20:43:07,052 INFO    MainThread:42208 [internal.py:handle_exit():156] Internal process exited
